pintos/userprog/process.c

```diff
@@ -1048,15 +1072,25 @@ load_segment (struct file *file, off_t ofs, uint8_t *upage,
 
 		/* TODO: lazy_load_segment에 정보를 전달하기 위해 aux를 설정한다. */
 		// [HERE] 2
-		void *aux = NULL;
-		if (!vm_alloc_page_with_initializer (VM_ANON, upage,
+		struct lazy_load_info *aux = malloc(sizeof(struct lazy_load_info));
```

Comment:
이거 나중에 해제 할 필요는 없나요 ?
```suggestion
		struct lazy_load_info *aux = malloc(sizeof(struct lazy_load_info));
```